peachjam/views/legislation.py

```diff
@@ -541,7 +540,7 @@ def get_template_names(self):
             ]
         return super().get_template_names()
 
-    @cached_property
+    @property
```

Comment:
this goes to the db and therefore is "expensive", so it's worth caching if we do it more than once.

Same for `provision_citation`.

Reply:
right, fixed